lib/generators/migration/migration_generator.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails/generators/active_record/migration/migration_generator'
+
+# Overwrites the default `ActiveRecord::Generators::MigrationGenerator` to
+# include our own spec file for each migration generated.
+#
+module ManageIQ::Schema
+  class MigrationGenerator < ActiveRecord::Generators::MigrationGenerator
+    # Using both `source_root` (standard) and `source_paths` here so that we
+    # can default to the rails sources for the `migration.rb` template, but
+    # then use our own for the `migration_spec.rb` file.
+    source_root ActiveRecord::Generators::MigrationGenerator.source_root
+    source_paths << File.expand_path('templates', __dir__)
+
+    # Overwrite the defaulted generated .namespace method so that this class is
+    # found as the `migration` class.
+    #
+    # Some Thor/Rails magic happeninghere ... but basically it allows this to
+    # overwrite and work properly
+    #
+    def self.namespace(name = nil)
+      return super if name
+
+      "migration"
+    end
+
+    def self.miq_schema_root
+      ManageIQ::Schema::Engine.root
+      # File.expand_path(File.join("..", "..", ".."), __dir__)
```

Comment:
I think these commented bits can go away now.

Reply:
F... I kept these as comments there in case using `Pathname` was going to be a problem, but forgot to remove them once the tests passed.

Thanks for the ping.  Resolving.